doc/.sphinx/spellingcheck.yaml

```diff
@@ -1,3 +1,4 @@
+jobs: 8
```

Comment:
why 8?

Reply:
That's the example they used in their announcement. ;)
Does another value make more sense?

Reply:
Could we maybe use something equivalent to `$$nproc`?

Reply:
we are using GH runners with 2 CPUs currently so 2 would probably be best.

Reply:
> $$nproc

How do I get that in the config file?